rest/blip_sync.go

```diff
@@ -743,7 +788,9 @@ func (bh *blipHandler) sendRevisionWithProperties(body db.Body, sender *blip.Sen
 	delete(body, db.BodyRev)
 	delete(body, db.BodyDeleted)
 
-	outrq.SetJSONBody(body)
+	if err := outrq.SetJSONBody(body); err != nil {
+		return err
```

Comment:
Have we validated that there aren't any cases where it was appropriate to ignore error and continue here?  (tombstones w/ no body maybe).  Would need to confirm before making this change - same for the sequence above.